config/default.json

```diff
@@ -75,6 +75,7 @@
         "UserStatusAwayTimeout": 300,
         "MaxChannelsPerTeam": 2000,
         "MaxNotificationsPerChannel": 1000,
+        "ConfirmNotificationsToChannel": true,
```

Comment:
I know it'll make the setting name a bit longer but we have a convention for prepending settings that enable or disable functionality with `Enable`. Would you be ok making this `EnableConfirmNotificationsToChannel`?

Reply:
Sure! I'll modify the PR ASAP :smiley: